pkg/models/meter.go

```diff
@@ -135,6 +135,7 @@ func WindowSizeFromDuration(duration time.Duration) (WindowSize, error) {
 }
 
 type Meter struct {
+	Namespace     string
```

Comment:
Should this be returned in the API? If so, it should be added to openapi. If not, `json:"-"` should be added, but I'm not sure that's a good idea.

Also, if this information is necessary, we should probably introduce a static namespace store.

Reply:
Let's discuss it here: https://github.com/openmeterio/openmeter/pull/323